src/V3String.cpp

```diff
@@ -174,6 +174,10 @@ string VString::replaceWord(const string& str, const string& from, const string&
     return result;
 }
 
+bool VString::startsWith(const string& str, const string& prefix) {
+    return str.rfind(prefix, 0) == 0;  // Faster than .find(_) == 0
+}
+
```

Comment:
Please pull this into a separate pre-patch and can push at least this can use it:

            src/V3OptionParser.cpp:128:            if (!std::strncmp(optp, act.first.c_str(), act.first.length())) {